testsuite/standalone/src/test/java/org/wildfly/extras/creaper/commands/orb/ChangeOrbOnlineTest.java

```diff
@@ -0,0 +1,232 @@
+package org.wildfly.extras.creaper.commands.orb;
+
+import static org.junit.Assert.assertEquals;
+
+import org.jboss.arquillian.container.test.api.ContainerController;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.junit.InSequence;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.dmr.ModelNode;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.wildfly.extras.creaper.commands.foundation.offline.ConfigurationFileBackup;
+import org.wildfly.extras.creaper.core.CommandFailedException;
+import org.wildfly.extras.creaper.core.ManagementClient;
+import org.wildfly.extras.creaper.core.ServerVersion;
+import org.wildfly.extras.creaper.core.offline.OfflineManagementClient;
+import org.wildfly.extras.creaper.core.offline.OfflineOptions;
+import org.wildfly.extras.creaper.core.online.ModelNodeResult;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.OnlineOptions;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.ReadResourceOption;
+import org.wildfly.extras.creaper.test.ManualTests;
+import com.google.common.collect.ImmutableMap;
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+
+@Category(ManualTests.class)
+@RunWith(Arquillian.class)
+@RunAsClient
```

Comment:
`@RunAsClient` is useless, please remove. The warnings from Arquillian that there's no running deployment are a bug, see [ARQ-2012](https://issues.jboss.org/browse/ARQ-2012).